src/main/java/edu/wisc/my/rssToJson/service/RsstoJsonServiceImpl.java

```diff
@@ -51,6 +65,20 @@ public JSONObject getJsonFromURL(String endpoint) {
         }
         jsonToReturn.put("items", entries);
         jsonToReturn.put("status", "ok");
+        //Clean undesirable unicode literals from the JsonObject
```

Comment:
Inline further for brevity?

Lines 68 through 78 reduce to
```
jsonStringToClean = jsonStringToClean.replaceAll("\\u2019", "'"); // right curly single quote becomes single quote
jsonStringToClean = jsonStringToClean.replaceAll("\\u2014", "-"); // en dash becomes minus sign
```

thereby eliminating need for `unicodeToClean`?

Like, the `replaceAll`s are two lines of code, they don't need to be represented as JSON?